### DISCUSSION TITLE
Problem with "psrp" CNAME with 2 postal server

### DISCUSSION THREAD

**Discussion by mascor** [postalserver/postal#2940] [Help with using Postal] [answered]
I have set up 2 postal servers:
- postal.mydomain.com
- postal2.mydomain.com

The DNS check shows a warning because it asks:
"There is a CNAME record at psrp.mydomain.com but it points to rp.postal.mydomain.com which is incorrect. It should point to rp.postal2.mydomain.com."
but I can set psrp.mydomain.com with only one domain, so the other one will fail.
Do you know how I can solve it?
Thanks

**Comment by willpower232** [comment]
It depends on what you're doing with the config file. The one in the config file should refer to Postal directly and then all domains you manage in the UI should have a psrp pointing at the domain in the config file.

It sounds like something is confused along the way.

**Comment by mascor** [comment]
In my postal file, I have:

dns:
  mx_records:
    - mx.postal2.mydomain.com
  spf_include: spf.postal2.mydomain.com
  return_path_domain: rp.postal2.mydomain.com
  route_domain: routes.postal2.mydomain.com
  track_domain: track.postal2.mydomain.com

in the /opt/postal/config/postal.yml there is no refer to psrp.mydomain.com so I cannot change it.

If I change this record: psrp.mydomain.com, which is pointing to rp.postal.mydomain.com, and "pass" the check on the first server, the control on the first server will fail like it's failing on the second server now.

Also in the documentation: https://docs.postalserver.io/getting-started/dns-configuration there is no refer to the psrp record.

**Reply by willpower232** [reply]
Oh you have set up two completely separate actual servers running Postal and then added the same domain to both?

I think you can change the prefix in one of the config files so one server uses psrp and the other uses something else.

**Reply by mascor** [reply]
Which prefix should I change?

This is my configuration
dns:
mx_records:
- mx.postal2.mydomain.com
spf_include: spf.postal2.mydomain.com
return_path_domain: rp.postal2.mydomain.com
route_domain: routes.postal2.mydomain.com
track_domain: track.postal2.mydomain.com

**Reply by willpower232** [reply, marked as answer]
this one

![image](https://github.com/postalserver/postal/assets/1619102/f2776b5f-4d04-4357-ac1a-979e31ab2607)
